automatic/keepass-plugin-readablepassphrasegen/update.ps1

```diff
@@ -25,16 +25,15 @@ function global:au_AfterUpdate {
 
 function global:au_GetLatest {
     $page = Invoke-WebRequest -Uri $releases -UseBasicParsing
-    $regexUrl = 'Version (?<version>[\d\.]+)'
+    $regexUrl = 'ReadablePassphrase\.(.*)\.plgx$'
```

Comment:
I like to use named index when looking at matches than numbered ones. Please see the previous code to use for that. Can you also fix the regular expression to match on . and digits only as the previous one did.